<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>637</width>
    <height>557</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>70</y>
     <width>301</width>
     <height>391</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border-bottom-right-radius:50px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>271</width>
     <height>391</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/image/background.jpg);
border-top-left-radius:50px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="isim">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>110</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgba(0,0,0,0);
border:none;
border-bottom:2px solid rgba(46,82,101,200);
color:rgbe(0,0,0,240);
padding-bottom:7px;</string>
   </property>
   <property name="placeholderText">
    <string>İsminiz</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="soyisim">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>145</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgba(0,0,0,0);
border:none;
border-bottom:2px solid rgba(46,82,101,200);
color:rgbe(0,0,0,240);
padding-bottom:7px;</string>
   </property>
   <property name="placeholderText">
    <string>Soyadınız</string>
   </property>
  </widget>
  <widget class="QPushButton" name="instagram">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>435</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Social Media Circled</family>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;</string>
   </property>
   <property name="text">
    <string>D</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>440</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Bilgi Almak için</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Cikis_buton">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>80</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"> background-color:rgba(150, 123, 111, 255);
</string>
   </property>
   <property name="text">
    <string>X</string>
   </property>
  </widget>
  <widget class="QLabel" name="Geri_bildirim">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>360</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:red;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Geri_donus">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>80</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"> background-color:rgba(150, 123, 111, 255);
</string>
   </property>
   <property name="text">
    <string>&lt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="kullanici_adi">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>180</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgba(0,0,0,0);
border:none;
border-bottom:2px solid rgba(46,82,101,200);
color:rgbe(0,0,0,240);
padding-bottom:7px;</string>
   </property>
   <property name="placeholderText">
    <string>Kullanıcı Adınız</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="mail">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>215</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgba(0,0,0,0);
border:none;
border-bottom:2px solid rgba(46,82,101,200);
color:rgbe(0,0,0,240);
padding-bottom:7px;</string>
   </property>
   <property name="placeholderText">
    <string>Mail Adresiniz</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="sifre">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>250</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgba(0,0,0,0);
border:none;
border-bottom:2px solid rgba(46,82,101,200);
color:rgbe(0,0,0,240);
padding-bottom:7px;</string>
   </property>
   <property name="placeholderText">
    <string>Şifreniz</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="dogum_tarih">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>285</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgba(0,0,0,0);
border:none;
border-bottom:2px solid rgba(46,82,101,200);
color:rgbe(0,0,0,240);
padding-bottom:7px;</string>
   </property>
   <property name="placeholderText">
    <string>Doğum Tarihi Ör: 02 02 2021</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ehliyet_tarih">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>320</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgba(0,0,0,0);
border:none;
border-bottom:2px solid rgba(46,82,101,200);
color:rgbe(0,0,0,240);
padding-bottom:7px;</string>
   </property>
   <property name="placeholderText">
    <string>Ehliyet Tarihi Ör: 02 02 2021</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Giris_buton">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>400</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"> background-color:rgba(150, 123, 111, 255);
</string>
   </property>
   <property name="text">
    <string>Kayıt Ol</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>60</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Kayıt Sayfası</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="Res.qrc"/>
 </resources>
 <connections/>
</ui>
